runtime/queries/matlab/locals.scm

```diff
@@ -0,0 +1,19 @@
+(function_definition name: (identifier) @local.definition ?) @local.scope
+(function_arguments (identifier)* @local.definition)
+
+(lambda (arguments (identifier) @local.definition)) @local.scope
+
+(assignment left: ((function_call
+                         name: (identifier) @local.definition)))
+(assignment left: ((struct . [(function_call
+                                    name: (identifier) @local.definition)
+                                  (identifier) @local.definition])))
```

Comment:
It looks like the query analysis is upset about this `struct` node. Was this renamed? I don't see it in the `grammar.js`

Reply:
I finally had the decency of copying the files over and running `tree-sitter test` before committing, so there should be no more errors like that. I'm sorry about this, it was a busy day and I'm more tired that I thought.